core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -694,6 +697,16 @@ public ServerBuilder meterRegistry(MeterRegistry meterRegistry) {
         return this;
     }
 
+    /**
+     * Sets a global naming rule for the name of services.
```

Comment:
Could you mention that this property can be overridden at virtual host or service level via `VirtualHostBuilder.defaultServiceNaming()` and `ServiceBindingBuilder.defaultServiceNaming()`?